test/cypress/integration/pages/obtener-respuestas.js

```diff
@@ -0,0 +1,28 @@
+import { ObtenerRespuestasBuscar } from '../../pages/obtener-respuestas/buscar';
+
+const buscar = new ObtenerRespuestasBuscar();
+
+describe( 'obtener-respuestas', () => {
+  describe( 'Buscar', () => {
+    it( 'should autocomplete results', () => {
+      buscar.open();
+      buscar.enter( 'divulgación de cierre' );
+      buscar.autocomplete().should( 'be.visible' );
+    } );
+
+    it( 'should return results', () => {
+      buscar.open();
+      buscar.enter( 'divulgación de cierre' );
+      buscar.search();
+      buscar.resultsSection().should( 'be.visible' );
+    } );
+
+    it( 'should correct spelling', () => {
+      buscar.open();
+      buscar.enter( 'vehíclo' );
+      buscar.search();
+      buscar.resultsHeader().contains( 'resultados para “vehículo”' );
+      buscar.resultsHeader().siblings( 'p' ).first().contains( 'Busca de' );
```

Comment:
These need to be assertions.

```suggestion
      buscar.resultsHeader().should( 'contain', 'resultados para “vehículo”' );
      buscar.resultsHeader().siblings( 'p' ).first().should( 'contain', 'Busca de vehíclo' );
```

Reply:
Done